master/shapes.html

```diff
@@ -256,7 +256,7 @@ <h2 id="EllipseElement">The <span class="element-name">'ellipse'</span> element<
 <p>
   The <a>'cx'</a> and <a>'cy'</a> coordinates define the center of the ellipse.
 </p>
-
+</bx
```

Comment:
*Woops!* How embarrassing. I must have introduced that error at the very last moment; clearly, I need to figure out how to introduce an XML validator into my workflow.

As this is an obvious and trivial mistake, I've force-pushed an amended commit.

Thanks!